### ISSUE TITLE
Missing installation instructions for Delphi last 7 versions

### ISSUE THREAD

**Issue by gwideman** [JAM-Software/Virtual-TreeView#899]
The current INSTALL.txt speaks of supporting up to Win 7 and 2008, and upto Delphi XE5. We're now 7 versions of Delphi down the road, and the last of those install instructions does not appear to work.

Ie: After translating the instructions to 10.3 IDE, and installing VirtualTreeView, when I then compile a project for a default VCL Win 32 project (or the provided Minimal demo) I get "Cannot resolve unit name 'VirtualTrees', and code insight marks uses... VirtualTrees as missing.

So maybe the XE5 instructions are missing a step that's now needed?  Here's the XE5 text:

> Delphi / RAD Studio XE2 - XE5 Installation
> ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
> 1. Open the project group "Packages\RAD Studio XE2\RAD Studio XE*.groupproj"
> 2. Right click on "VirtualTreesD1*.bpl" and click "Install"
> 3. Go to "Tools > Options > Environment Options > Delphi Options > Library > 
> Library Path > [...]"
>   Browse to the "Source" folder of VirtualTreeView, press "OK", "Add", "OK", "OK"

... which I translated to:

> Delphi / RAD Studio 10.3 Installation
> ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
> 1. File > Open the project group "...\Packages\RAD Studio 10.3\VirtualTreeView.groupproj"
> 2. Right click on "VirtualTreesD26.bpl" and click "Install"
> 3. Go to "Tools > Options > Language > Delphi Options > Library > 
> Library Path > [...]"
>   Browse to the "Source" folder of VirtualTreeView, press "OK", "Add", "OK", "Save"

As I said, the Install step went smoothly. The TVirtualXxx components end up on the component palette, can be added to a form, and add their library to the uses clause. But they are not found by the compile step, nor code insight functions.

Now, a possible clue:  In the Minimal project, if I add Win64 as a platform and set it active, now Delphi can build the project (well, after deleting Icon_MainIcon instruction from the proj file).  And it runs OK.  But switching bck to Win32 platform, still can't resolve VirtualTrees.

This odd behavior seems to conflict with VirtualTreeView.groupproj, where VirtualTreesD25 is set to compile for Win 32.

Bottom line, the INSTALL.txt instructions as they currently stand don't lead to a successful install.

For what it's worth, this is a freshly installed Delphi 10.3 CE on Windows 7-64 with all updates.

I also note that this case seems parallel (but reverse details) to this other case from 5 months ago:
https://github.com/Virtual-TreeView/Virtual-TreeView/issues/847

**Comment by joachimmarder** [comment]
> Now, a possible clue: In the Minimal project, if I add Win64 as a platform and set it active, now Delphi can build the project

In that case you've most likely modified the Win64 search path, not the Win32 one.  I improved the instructions.

**Comment by joachimmarder** [comment]
> The current INSTALL.txt speaks of supporting up to Win 7 and 2008, and upto Delphi XE5.

Where did you get that from? You must have an old version of the Install.txt. 
Check here: https://github.com/Virtual-TreeView/Virtual-TreeView/commits/master/INSTALL.txt

**Comment by gwideman** [comment]
a) You're absolutely right.  With multiple versions of TVT installed for different versions of Delphi, I somehow was viewing the wrong INSTALL file.  Grrr, very sorry about that.
b) Your just-committed install text. Improved in that it points out the necessity to build VirtualTrees* for both Win 32 and 64, and mentions "the dropdown box" (the Selected Platform dropdown) but...
c)  The menu path to do so is different in 10.3, and perhaps earlier. Not:
"Tools > Options > Environment Options > Delphi Options > Library > Library Path > [...]"
but rather...
"Tools > Options > Language > Delphi Options > Library > Library Path > [...]"

Thanks for your attention.

**Comment by joachimmarder** [comment]
Although the differences are quite small, I now made an own section for 10.3 and higher
